src/Map.vue

```diff
@@ -938,7 +943,7 @@ export default {
       showGrid: this.fromStorage("showGrid", "true") === "true",
       showHeightmap: this.fromStorage("showHeightmap", "false") === "true",
       showLocations: this.fromStorage("showLocations", "false") === "true",
-      selectedMap: this.fromStorage("selectedMap", undefined), // selected map in top selector
+      selectedMap: this.initialMap || this.fromStorage("selectedMap", undefined), // selected map in top selector
```

Comment:
I think this `|| this.fromStorage("selectedMap", undefined)` can also be removed... It *should* be doing nothing because `initialMap` should always be provided by the router.